[2024-09-26 21:54:56,710] INFO _log() L97    * Restarting with stat - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:01,143] INFO _log() L97    * Detected change in 'C:\\Projetos\\API-Doces-Simplificada\\schemas\\produto.py', reloading - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:01,357] INFO _log() L97    * Restarting with stat - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:05,163] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:55:05] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:05,263] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:55:05] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:05,266] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:55:05] "GET /openapi/static/css/swagger-ui.css HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:05,267] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:55:05] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:05,396] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:55:05] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:21,512] WARNING get_produto() L88   Erro ao buscar produto '1', Produto não encontrado na base :/ - call_trace=C:\Projetos\API-Doces-Simplificada\app.py L88  
[2024-09-26 21:55:21,513] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:55:21] "[31m[1mGET /produto?id=1 HTTP/1.1[0m" 400 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:29,435] WARNING get_produtos() L108  Erro ao buscar por lista de produtos. Produto não encontrado na base :/ - call_trace=C:\Projetos\API-Doces-Simplificada\app.py L108 
[2024-09-26 21:55:29,436] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:55:29] "[31m[1mGET /produtos HTTP/1.1[0m" 400 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:55:59,469] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:55:59] "POST /produto HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:56:24,502] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:56:24] "GET /produto?id=1 HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:56:34,580] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:56:34] "GET /produtos HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:57:04,374] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:57:04] "GET /produtos HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:57:43,494] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:57:43] "POST /produto HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:58:04,628] WARNING add_produto() L67   Erro ao adicionar produto 'Docinho de Festa', Produto de mesmo nome já salvo na base :/ - call_trace=C:\Projetos\API-Doces-Simplificada\app.py L67  
[2024-09-26 21:58:04,629] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:58:04] "[31m[1mPOST /produto HTTP/1.1[0m" 409 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:58:56,360] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:58:56] "POST /produto HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:59:30,881] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:59:30] "POST /produto HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:59:40,864] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:59:40] "POST /produto HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 21:59:46,865] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 21:59:46] "POST /produto HTTP/1.1" 200 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 22:00:01,292] ERROR log_exception() L838  Exception on /produto [DELETE] - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py L838 
Traceback (most recent call last):
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask_cors\extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\app.py", line 123, in del_produto
    produto_nome = query.nome
                   ^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\pydantic\main.py", line 856, in __getattr__
    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')
AttributeError: 'ProdutoBuscaSchema' object has no attribute 'nome'
[2024-09-26 22:00:01,309] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 22:00:01] "[35m[1mDELETE /produto?id=6 HTTP/1.1[0m" 500 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 22:00:30,233] INFO _log() L97    * Detected change in 'C:\\Projetos\\API-Doces-Simplificada\\app.py', reloading - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 22:00:30,455] INFO _log() L97    * Restarting with stat - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 22:00:32,282] ERROR log_exception() L838  Exception on /produto [DELETE] - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py L838 
Traceback (most recent call last):
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask_cors\extension.py", line 194, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\app.py", line 123, in del_produto
    produto_nome = query.nome
                   ^^^^^^^^^^
  File "C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\pydantic\main.py", line 856, in __getattr__
    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')
AttributeError: 'ProdutoBuscaSchema' object has no attribute 'nome'
[2024-09-26 22:00:32,287] INFO _log() L97   127.0.0.1 - - [26/Sep/2024 22:00:32] "[35m[1mDELETE /produto?id=6 HTTP/1.1[0m" 500 - - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
[2024-09-26 22:00:40,098] INFO _log() L97    * Detected change in 'C:\\Projetos\\API-Doces-Simplificada\\app.py', reloading - call_trace=C:\Projetos\API-Doces-Simplificada\env\Lib\site-packages\werkzeug\_internal.py L97  
